src/api/app/models/bs_request.rb

```diff
@@ -73,7 +73,22 @@ class BsRequest < ApplicationRecord
   has_many :bs_request_actions, dependent: :destroy
   has_many :reviews, dependent: :delete_all
   has_many :comments, as: :commentable, dependent: :destroy
-  has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
+  # NOTE: The "where" is forcing the STI to load all possible inheritance
```

Comment:
@danidoni looks good to me, I just find the comment confusing. It sounds like it does not work with the `where`.

Reply:
Mmm, let me rewrite it

Reply:
I went for another path, please check it out